var/spack/repos/builtin/packages/rocalution/package.py

```diff
@@ -16,20 +16,22 @@ class Rocalution(CMakePackage):
        other scientific software packages."""
 
     homepage = "https://github.com/ROCmSoftwarePlatform/rocALUTION"
-    url      = "https://github.com/ROCmSoftwarePlatform/rocALUTION/archive/rocm-3.5.0.tar.gz"
+    url      = "https://github.com/ROCmSoftwarePlatform/rocALUTION/archive/rocm-3.7.0.tar.gz"
 
     maintainers = ['srekolam', 'arjun-raj-kuppala']
 
+    version('3.7.0', sha256='4d6b20aaaac3bafb7ec084d684417bf578349203b0f9f54168f669e3ec5699f8')
     version('3.5.0', sha256='be2f78c10c100d7fd9df5dd2403a44700219c2cbabaacf2ea50a6e2241df7bfe')
 
-    depends_on('cmake@3.5.2', type='build')
-    for ver in ['3.5.0']:
+    depends_on('cmake@3.5:', type='build')
+    for ver in ['3.5.0', '3.7.0']:
         depends_on('hip@' + ver,  when='@' + ver)
         depends_on('rocblas@' + ver, type='build', when='@' + ver)
```

Comment:
can you check the dependency for rocblas, rocsparse, rocprim . i think type=link should suffice.